toolchains/local_java_repository.bzl

```diff
@@ -204,6 +204,8 @@ def _local_java_repository_impl(repository_ctx):
 
     # Detect version
     version = repository_ctx.attr.version if repository_ctx.attr.version != "" else _detect_java_version(repository_ctx, java_bin)
+    if version == None:
+        return
```

Comment:
Since this happens in the implementation function of a repository rule, we need to generate something or the user will just get errors about missing targets. Could you do something similar to line 194 here?

Reply:
I used the template from line `194` to indicate an error. The output is still somewhat verbose, though.

```
ERROR: /home/norbert/.cache/bazel/_bazel_norbert/88cb78e88cb9acdaa4bfce98a60561bf/external/rules_java~7.3.2~toolchains~local_jdk/BUILD.bazel:2:10: in fail_rule rule @@rules_java~7.3.2~toolchains~local_jdk//:jdk:
Traceback (most recent call last):
        File "/home/norbert/.cache/bazel/_bazel_norbert/88cb78e88cb9acdaa4bfce98a60561bf/external/rules_java~7.3.2/toolchains/fail_rule.bzl", line 19, column 13, in _fail_rule_impl
                fail("%s %s" % (ctx.attr.header, ctx.attr.message))
Error in fail: Auto-Configuration Error: Cannot detect Java version of bin/java in /home/norbert/.asdf/installs/java/openjdk-21; make sure it points to a valid Java executable
ERROR: /home/norbert/.cache/bazel/_bazel_norbert/88cb78e88cb9acdaa4bfce98a60561bf/external/rules_java~7.3.2~toolchains~local_jdk/BUILD.bazel:2:10: Analysis of target '@@rules_java~7.3.2~toolchains~local_jdk//:jdk' failed
```